packages/contracts/test/plugins/utils/addresslist.ts

```diff
@@ -49,54 +49,61 @@ describe('AddresslistMock', function () {
 
   context('addresslistLengthAtBlock', function () {
     it('returns the right length after addresses were added', async () => {
-      let tx1 = await addresslist.addAddresses([signers[0].address]);
+      const tx1 = await addresslist.addAddresses([signers[0].address]);
       await ethers.provider.send('evm_mine', []);
 
-      let tx2 = await addresslist.addAddresses([
+      const tx2 = await addresslist.addAddresses([
         signers[1].address,
         signers[2].address,
       ]);
       await ethers.provider.send('evm_mine', []);
 
+      const rc1 = await tx1.wait();
+      const rc2 = await tx2.wait();
+
       expect(
-        await addresslist.addresslistLengthAtBlock(tx1.blockNumber - 1)
+        await addresslist.addresslistLengthAtBlock(rc1.blockNumber - 1)
       ).to.equal(0);
       expect(
-        await addresslist.addresslistLengthAtBlock(tx1.blockNumber)
+        await addresslist.addresslistLengthAtBlock(rc1.blockNumber)
       ).to.equal(1);
       expect(
-        await addresslist.addresslistLengthAtBlock(tx2.blockNumber)
+        await addresslist.addresslistLengthAtBlock(rc2.blockNumber)
       ).to.equal(3);
     });
 
     it('returns the right length after addresses were removed', async () => {
-      let tx1 = await addresslist.addAddresses([
+      const tx1 = await addresslist.addAddresses([
         signers[0].address,
         signers[1].address,
         signers[2].address,
       ]);
       await ethers.provider.send('evm_mine', []);
 
-      let tx2 = await addresslist.removeAddresses([signers[0].address]);
+      const tx2 = await addresslist.removeAddresses([signers[0].address]);
       await ethers.provider.send('evm_mine', []);
 
-      let tx3 = await addresslist.removeAddresses([
+      const tx3 = await addresslist.removeAddresses([
         signers[1].address,
         signers[2].address,
       ]);
       await ethers.provider.send('evm_mine', []);
 
-      expect(tx1.blockNumber).to.be.lt(tx2.blockNumber);
-      expect(tx2.blockNumber).to.be.lt(tx3.blockNumber);
+      const rc1 = await tx1.wait();
+      const rc2 = await tx2.wait();
+      const rc3 = await tx3.wait();
```

Comment:
could be shortened to
```suggestion
      const [rc1, rc2, rc3] = await Promise.all([tx1.wait(), tx2.wait(), tx3.wait()]);
```